qemu/tests/blockdev_inc_backup_add_bitmap_to_raw.py

```diff
@@ -0,0 +1,43 @@
+from virttest.qemu_monitor import QMPCmdError
+
+from provider.blockdev_live_backup_base import BlockdevLiveBackupBaseTest
+
+
+class BlockdevIncbkAddBitmapToRawImgNeg(BlockdevLiveBackupBaseTest):
+    """Negative test: Add bitmaps to a raw image"""
+
+    def prepare_test(self):
+        self.preprocess_data_disks()
+        self.prepare_main_vm()
+
+    def add_bitmap_to_raw_image(self):
+        try:
+            kargs = {'node': self._source_nodes[0],
+                     'name': self._bitmaps[0],
+                     'persistent': self._full_backup_options['persistent']}
+            self.main_vm.monitor.block_dirty_bitmap_add(**kargs)
+        except QMPCmdError as e:
+            error_msg = self.params['error_msg'].format(
+                node=self._source_nodes[0])
+            if error_msg not in str(e):
+                self.test.fail('Unexpected error: %s' % str(e))
```

Comment:
```suggestion
                self.test.fail('Unexpected error: %s' % str(e))
         else:
             self.test.error('No catched expected QMPCmdError.')       
```
I am afraid that `add_bitmap_to_raw_image` will not work in negative testing in case there is no `QMPCmdError`.

Reply:
yes, yongxue, u r right, I just updated the case